apps/sms/test/marionette/lib/views/thread.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+var View = require('./view');
+
+function Thread() {
+  View.apply(this, arguments);
+}
+module.exports = Thread;
+
+Thread.prototype = {
+  __proto__: View.prototype,
+
+  callContact: function() {
+    this.pageAccessors.Thread.callButton.tap();
+
+    var Dialer = require('./dialer');
```

Comment:
I think it's test responsibility to load Dialer libs, and switch to dialer so that view doesn't know about it. In reality it doesn't know either - it just issues activity request and in theory any supported app can handle this requested.